src/engine/test/source/metrics/otl_signals_test.cpp

```diff
@@ -34,43 +35,37 @@ class MetricsOtlMeterTest : public ::testing::Test
 
 TEST_F(MetricsOtlMeterTest, invalidValueCounter)
 {
-    Metrics::instance().initMetrics("example", INPUT_PATH);
+    // TODO: Resolve fullpath accordingly.
+    Metrics::instance().initMetrics("engine-metrics", INPUT_PATH);
     EXPECT_ANY_THROW(Metrics::instance().addCounterValue("Sockets", -1UL));
 }
 
-TEST_F(MetricsOtlMeterTest, repeatedInit)
```

Comment:
Why removed?

Reply:
because to instantiate it you need the INPUT_PATH that should be defined in the main of the tests.